src/ArborX_LinearBVH.hpp

```diff
@@ -125,22 +125,24 @@ template <typename DeviceType>
 template <typename Primitives>
 BoundingVolumeHierarchy<DeviceType>::BoundingVolumeHierarchy(
     Primitives const &primitives)
```

Comment:
What is the reasoning for the term `Primitives` here?

Reply:
It is a generic way of qualifying the elements you are indexing.  I like it better than "geometrical objects".